tests/rector.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Caching\ValueObject\Storage\FileCacheStorage;
+use Rector\Config\RectorConfig;
+
+return RectorConfig::configure()
+    ->withCache(
+        cacheClass: FileCacheStorage::class,
+        cacheDirectory: __DIR__ . '/../.rector'
+    )->withPaths([
+        __DIR__ . '/../src',
+        __DIR__ . '/../tests',
+    ])
+    ->withTypeCoverageLevel(0)
+    ->withDeadCodeLevel(0)
+    ->withCodeQualityLevel(0);
```

Comment:
Why not go as far as we can without having to change anything? :) 
```suggestion
    ->withTypeCoverageLevel(0)
    ->withDeadCodeLevel(6)
    ->withCodeQualityLevel(21);
```